static/app/views/explore/charts/confidenceFooter.tsx

```diff
@@ -39,7 +47,7 @@ function confidenceMessage({
   const sampleCountComponent = <Count value={sampleCount} />;
   if (confidence === 'low') {
     if (isTopN) {
-      if (dataScanned === 'partial') {
+      if (dataScanned === 'partial' && !normalSamplingModeEnabled) {
```

Comment:
might be a good idea to just store `dataScanned === 'partial' && !normalSamplingModeEnabled` in it's own variable to limit repitition.